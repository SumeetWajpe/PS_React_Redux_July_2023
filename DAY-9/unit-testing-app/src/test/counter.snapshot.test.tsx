import renderer from "react-test-renderer";
import Counter from "../counter/counter";

it("renders correctly", () => {
  const tree = renderer.create(<Counter initialCount={100} />).toJSON();
  expect(tree).toMatchSnapshot();
});
