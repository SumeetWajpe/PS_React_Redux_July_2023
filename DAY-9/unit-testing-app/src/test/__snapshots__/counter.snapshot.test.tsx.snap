// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
   
  <div>
    <p>
      Count : 
      100
    </p>
     
    <button
      onClick={[Function]}
    >
      +++ 
    </button>
     
  </div>
</div>
`;
